docs/upgrade/xenial_backup_install_restore.rst

```diff
@@ -0,0 +1,284 @@
+ ``~/Persistent/app_service/site-specific`` file on the *Admin Workstation*.
+
+Step 3: Set up SSH access to the servers
+----------------------------------------
+
+Once you have installed Xenial on the servers, you'll need to configure
+key-based SSH access to the servers from the *Admin Workstation*.
+
+First, connect the *Admin Workstation* directly to the firewall via Ethernet,
+and start up the *Admin Workstation* with persistence enabled and an
+administration password set. You may need to update the network settings in
+Tails to use the static IP that was set up during the firewall configuration for
+the *Admin Workstation*.
+
+Then, remove the previous instance's connection configuration from the *Admin
+Workstation*, by opening a terminal and running the following commands:
+
+.. code:: sh
+
+ rm ~/.ssh/{config,known_hosts}
+ rm ~/Persistent/securedrop/install_files/ansible-base/app-source-ths
+ rm ~/Persistent/securedrop/install_files/ansible-base/app-*-aths
+ rm ~/Persistent/securedrop/install_files/ansible-base/mon-*-aths
+
+Next, copy the *Admin Workstation*'s SSH public key to the new servers. To do
+so, you will need the IP addresses of the servers, and the username and password
+of the administrator account created during the OS installation. The commands
+below use the default values, but make sure to substitute your own:
+
+.. code:: sh
+
+ # copy key to app
+ ssh-copy-id sdadmin@10.20.2.2
+
+ # copy key to mon
+ ssh-copy-id sdadmin@10.20.3.2
+
+You will be prompted by both commands for the shell account password.
+
+To confirm that key-based SSH access is working, run the following commands:
+
+.. code:: sh
+
+  ssh sdadmin@10.20.2.2 hostname
+  ssh sdadmin@10.20.3.2 hostname
+
+In both cases, the commands should return the hostname of the remote server,
+without requiring a password.
+
+
+Step 4: Install SecureDrop
+--------------------------
+
+Once you have set up SSH access, it's time to install SecureDrop. As most of the
+application settings are preserved on the *Admin Workstation* from your previous
+instance, this process will be simpler than your first installation.
+
+First, you'll need make sure your *Admin Workstation*'s SecureDrop application
+code is up-to-date and validated. From a terminal, run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ git checkout 0.12.0
+ git tag -v 0.12.0
+
+You should see ``Good signature from "SecureDrop Release Signing Key"`` in the
+output of that last command, along with the fingerprint ``"2224 5C81 E3BA EB41
+38B3 6061 310F 5612 00F4 AD77"``
+
+.. caution::
+
+ If you do not, signature verification has failed and you should not proceed
+ with the installation. If this happens, please contact us at
+ securedrop@freedom.press.
+
+If the command above returns the expected value, you may proceed with the installation.
+
+First, run the following command to set up the SecureDrop administration environment:
+
+.. code:: sh
+
+  ./securedrop-admin setup
```

Comment:
For users that have not updated Tails in a while, there will be the install once/install each time prompt when downloading apt package (described in https://docs.securedrop.org/en/release-0.11.0/install.html). Since either option will work, might not be worth mentioning here?

Reply:
gonna leave it out to keep the doc clean